main.go

```diff
@@ -303,6 +308,18 @@ func getDbHandle(db *mysql.Conn) (*mysql.Conn, error) {
 	return mysql.Connect(net.JoinHostPort(mysqlHost, mysqlPort), mysqlUser, mysqlPassword, mysqlDatabase)
 }
 
+func getRiemannHandle(riemann *raidman.Client) (*raidman.Client, error) {
+	if riemann != nil {
+		if _, err := riemann.Query(`service =~ "riemann %"`); err != nil {
+			return nil, err
```

Comment:
I think you should Close the client here.

Reply:
I'm not sure this is necessary, since we're likely to be in this case if the client connection is broken and what the `raidman.Client.Close()` method does is to close the connection (which is already closed): https://github.com/amir/raidman/blob/master/raidman.go#L340